pkg/service/info.go

```diff
@@ -52,6 +52,7 @@ type Protocols struct {
 type NodeRecommendation struct {
 	Network      string `json:"network" header:"network"`
 	NodeSpec     string `json:"nodeSpec" header:"nodeSpec"`
+	NodeSpecMultiplier int      `json:"nodeSpecMultiplier" header:"NodeSpecMultiplier"`
```

Comment:
```suggestion
	NodeSpecMultiplier int      `json:"nodeSpecMultiplier" header:"nodeSpecMultiplier"`
```
to align with other fields

Reply:
@NihalKPatel don't forget here.